python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -163,15 +187,18 @@ def move_while_exposing(self, value):
         self.assert_corrections(enabled=False)
 
         # FIXME: Get position from telescope if elevation = 0.
-        azimuth = id_data.data.azimuth
+        azimuth = id_data.data.azimuth % 360.
         elevation = id_data.data.elevation
 
         if elevation == 0.:
             # Get telescope position. Will flush data stream in order to get the most updated position
-            position = await self.mcs.tel_mountEncoders.next(flush=True,
-                                                             timeout=self.cmd_timeout)
-            azimuth = position.azimuthCalculatedAngle
-            elevation = position.elevationCalculatedAngle
+            position = await self.ptg.tel_currentTargetStatus.next(flush=True,
+                                                                   timeout=self.cmd_timeout)
+            # These values comes as +DD:MM:SS.SS strings from the pointing component. Need to parse them
+            # to floats here.
+            # wrapping azimuth between 0 and 360, ataos dont care about wrap.
```

Comment:
I find the comment "ataos dont care about wrap" a bit confusing. I think I know what you mean, but it sounds at first as if it means you don't need to bother to wrap it at all (and then you wrap the angle). Consider dropping that bit since I don't think it adds useful information.